profiler/src/ProfilerEngine/Datadog.Profiler.Native/FrameStore.cpp

```diff
@@ -501,7 +502,10 @@ std::pair<std::string, mdTypeDef> FrameStore::GetMethodName(
     auto [methodName, mdTokenType] = GetMethodNameFromMetadata(pMetadataImport, mdTokenFunc);
     if ((methodName.empty()) || (genericParametersCount == 0))
     {
-        return std::make_pair(std::move(methodName), mdTokenType);
+        // get the method signature
+        std::string signature = GetMethodSignature(_pCorProfilerInfo, pMetadataImport, functionId, mdTokenFunc);
+
+        return std::make_pair(std::move(methodName + signature), mdTokenType);
```

Comment:
no need to use `std::move` header, the `+` operator will produce a rvalue reference. (example l.549)

Reply:
✅ thanks